ride/swap.ride

```diff
@@ -0,0 +1,196 @@
+{-# STDLIB_VERSION 6 #-}
+{-# CONTENT_TYPE DAPP #-}
+{-# SCRIPT_TYPE ACCOUNT #-}
+
+#-----------------
+# GLOBAL VARIABLES
+#-----------------
+
+
+let SEP = "__"
+
+let feeScale = 100_000.toBigInt()
+
+func getStringOrFail(key: String)   = {this.getString(key).valueOrErrorMessage("No data for this.key=" + key)}
+
+let poolDisabled = "Pool disabled".throw()
+let poolNotExist = "Pool is not exist".throw()
+let slippageTooBig = "Slippage is too big".throw()
+
+func keyManagerPublicKey() = "%s__managerPublicKey"
+func keyPendingManagerPublicKey() = "%s__pendingManagerPublicKey"
+
+func fc() = {"%s__factoryContract"}
+let factoryContract = addressFromStringValue(getStringOrFail(fc()))
+
+func feeLiquidityContractAddress() = ["%s", "feeLiquidityContractAddress"].makeString(SEP)
+func feeLiquidity() = ["%d", "feeLiquidity"].makeString(SEP)
```

Comment:
`%s` should be used. This info describes the key's structure.

Reply:
agree. corrected